recipe/meta.yaml

```diff
@@ -87,7 +90,8 @@ outputs:
         - cam -l
```

Comment:
This likely isn’t getting run at any time. You should move it to the top section

Reply:
wait, what? If I move it to the top section it should get run for each output, which I don't want, because it's not in the rpi variant....what am I missing..?

Reply:
What does the cam script do? Why did the rpi folk decide to remove it?

Reply:
honestly, at the moment I don't remember...I'm looking into it.

But, does it really matter?

Reply:
its good to understand a little bit why things are different, document that you acknlodge it, and provide a path for others to improve on it without asking you "do you remember why you did this strange thing".